admin/resources/js/helpers/routes.ts

```diff
@@ -0,0 +1,36 @@
+export const classificationTable = (locale: string): string =>
```

Comment:
I like the idea of this file, though it would seem like this code should somehow be combined with the [Routes in the PoolDashboard component](https://github.com/GCTC-NTGC/gc-digital-talent/blob/feature/form-submission-toast/admin/resources/js/components/PoolDashboard.tsx#L68) so that there aren't multiple sources of truth for the routes. Thoughts? Also, where is the argument `locale` used for?